peps/pep-0768.rst

```diff
@@ -136,20 +136,20 @@ A new structure is added to PyThreadState to support remote debugging:
 
     typedef struct _remote_debugger_support {
         int debugger_pending_call;
-        char debugger_script[MAX_SCRIPT_SIZE];
+        char debugger_script_path[MAX_SCRIPT_PATH_SIZE];
+        char *debugger_buffer;
```

Comment:
You resolved my comment, but without addressing it. I think this line should be removed; `debugger_buffer` isn't referenced anywhere else. `debugger_script_path` is the buffer we intend to keep.

Reply:
Yeah sorry I think there was some missing changes in my amend :S